prime-factors/prime_factors.py

```diff
@@ -1,2 +1,10 @@
 def prime_factors(natural_number):
-    pass
+    list = []
```

Comment:
@zarniak Oh, no! Built-in method overwrite 🚓 !!

There are several functions and keywords already defined in Python and `list` is one of them. So it's better to find an alternative name for things you define. Usually appending an `_` is enough i.e. `list_`. And maybe something more descriptive would be even better. It's easy to see what a variable called `list_` holds in such a short snippet but it becomes hard once the code complexity grows. But maybe a bit more descriptive name would be even better? It's good when the code is "self-documenting". I do know that it's hard to come up with a good name :trollface: As a wise man said: "There are 2 hard problems in computer science: cache invalidation, naming things, and off-by-1 errors." ;)

Fortunately that built-in overwrite happens only in that function's scope:
```
>>> list('abc')
['a', 'b', 'c']
>>> def foobar():
...     list = 1
...     list('abc')
...
>>> list('abc')
['a', 'b', 'c']
>>> foobar()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 3, in foobar
TypeError: 'int' object is not callable
```